app/main/controller/image_ocr_controller.py

```diff
@@ -39,9 +49,8 @@ def post(self):
         if not texts:
             return
 
-        res_dict = MessageToDict(response, preserving_proto_field_name=True)
         app.logger.info(
```

Comment:
Let's enclose this logging line in a `try...except` block. The `except` can just be a `pass`

Reply:
I added a logging of the exception